source/jormungandr/jormungandr/scenarios/ridesharing/instant_system.py

```diff
@@ -65,6 +65,8 @@ def __init__(
         rating_scale_min=None,
         rating_scale_max=None,
         timeout=2,
+        crowfly_radius=200,
+        timeframe_duration=30,
```

Comment:
It would be better to have seconds as for almost anything else (and use them all the way if you can do `PT900S`).
Maybe rename it if you have to stay to minutes?